tools/topology/sof/pipe-dai-sched-playback.m4

```diff
@@ -0,0 +1,24 @@
+# DAI Playback connector
+
```

Comment:
Please add some comments here to describe the topology.

Reply:
> Please add some comments here to describe the topology.

Done